behave/features/steps/specialist_provider.py

```diff
@@ -26,9 +28,19 @@ def step_impl(context):
     compare_client_details_with_backend(context, case_id, client_section)
 
 
-@step(u'I select a case from the dashboard')
+@step(u'I select a case to accept from the dashboard')
 def step_select_special_provider_case(context):
     case_reference = CLA_SPECIALIST_CASE_TO_ACCEPT
+    select_a_case(context, case_reference)
+
+
+@step(u'I select a case to reject from the dashboard')
+def step_impl(context):
+    case_reference = CLA_SPECIALIST_CASE_TO_REJECT
+    select_a_case(context, case_reference)
+
+
+def select_a_case(context, case_reference):
 
     table = context.helperfunc.driver().find_element_by_css_selector(".ListTable")
     # this will only return a link if the case hasn't already been accepted
```

Comment:
Need to think through if this is ok here as it only looks for cases that haven't already been accepted.I assume this is right but want to check.